library/NotificationCenter/MessageDraft/EmailMessageDraft.php

```diff
@@ -165,6 +165,18 @@ public function getTextBody()
     public function getHtmlBody()
     {
         if ($this->objLanguage->email_mode == 'textAndHtml') {
+            // Switch the email template language for the notification
+            if ($GLOBALS['TL_LANGUAGE'] !== $this->getLanguage()) {
+                $GLOBALS['TL_LANGUAGE'] = $this->getLanguage();
+            }
+            if (interface_exists(\Symfony\Contracts\Translation\TranslatorInterface::class)
+                || interface_exists(\Symfony\Component\Translation\TranslatorInterface::class)) {
+                $translator = \Contao\System::getContainer()->get('translator');
```

Comment:
I don't think you should check for the interface, as the interface could be present even in Contao 3 (through other dependencies).

Reply:
I moved the switch to the custrutor of Email incl version check instead of interface check